pydarn/plotting/acf.py

```diff
@@ -253,104 +250,89 @@ def plot_acfs(cls, dmap_data: List[dict], beam_num: int = 0,
             ax = fig.add_subplot(gs[0, :])
             ax_pwr = fig.add_subplot(gs[1, 0])
             ax_phs = fig.add_subplot(gs[1, 1])
+            axes = [ax, ax_pwr, ax_phs]
+        else:
+            if ax is not None:
+                ax = ax
+            else:
+                ax = plt.gca()
+            axes = [ax]
+
+        real_dict = ACF.plot_acf_param(lags, masked_re, ax,
+                                       plot_blank=plot_blank,
+                                       marker='o',
+                                       color=real_color,
+                                       blank_marker=blank_marker,
+                                       **kwargs)
+        imag_dict = ACF.plot_acf_param(lags, masked_im, ax,
+                                       plot_blank=plot_blank,
+                                       marker='o',
+                                       color=imaginary_color,
+                                       blank_marker=blank_marker,
+                                       **kwargs)
+        # Set parameters (axis labels, line labels, etc.) of main plot
+        ax.set_ylabel(parameter)
+        ax.set_xlabel('Lag Number')
+        real_dict['data']['line'].set_label('Real')
+        imag_dict['data']['line'].set_label('Imag')
+        if plot_blank:
+            real_dict['data']['blank_line'].set_label('Real Blanked')
+            imag_dict['data']['blank_line'].set_label('Imag Blanked')
+            lim_val = 1.1 * max(np.nanmax(np.abs(re)), np.nanmax(np.abs(im)))
+        else:
+            lim_val = 1.1 * max(np.nanmax(np.abs(masked_re), np.nanmax(np.abs(masked_im))))
```

Comment:
This line throws a: `TypeError: 'numpy.float64' object cannot be interpreted as an integer`
when I use plot_blank=False

:(

```
rtn = pydarn.ACF.plot_acfs(data, beam_num=5, gate_num=12, normalized=True,
                start_time=datetime(2018, 5, 20, 9, 0), pwr_and_phs=True,
                plot_blank=False)
```

Reply:
Also sorry I've tried to figure out why it's doing this but I dunno

Reply:
```suggestion
            lim_val = 1.1 * max(np.nanmax(np.abs(masked_re),
                                            np.nanmax(np.abs(masked_im))))
```

And the line is too long style wise 👀

Reply:
Found and fixed the bug! Trying to uses nanmax on a masked array is not a good idea, by default masked_array.max() ignores masked values and all the NaN's were already masked.